[2022-07-29 11:41:35,899] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag_with_python_operator.get_name scheduled__2021-11-25T00:00:00+00:00 [queued]>
[2022-07-29 11:41:35,918] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag_with_python_operator.get_name scheduled__2021-11-25T00:00:00+00:00 [queued]>
[2022-07-29 11:41:35,918] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-29 11:41:35,918] {taskinstance.py:1357} INFO - Starting attempt 1 of 6
[2022-07-29 11:41:35,918] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-29 11:41:35,938] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): get_name> on 2021-11-25 00:00:00+00:00
[2022-07-29 11:41:35,942] {standard_task_runner.py:52} INFO - Started process 6003 to run task
[2022-07-29 11:41:35,945] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_with_python_operator', 'get_name', 'scheduled__2021-11-25T00:00:00+00:00', '--job-id', '554', '--raw', '--subdir', 'DAGS_FOLDER/CreateDagWithPythonOperator.py', '--cfg-path', '/tmp/tmpu8knf0z2', '--error-file', '/tmp/tmppeir_q2d']
[2022-07-29 11:41:35,946] {standard_task_runner.py:80} INFO - Job 554: Subtask get_name
[2022-07-29 11:41:36,020] {task_command.py:369} INFO - Running <TaskInstance: dag_with_python_operator.get_name scheduled__2021-11-25T00:00:00+00:00 [running]> on host 9f0f006fc89d
[2022-07-29 11:41:36,119] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=yosr
AIRFLOW_CTX_DAG_ID=dag_with_python_operator
AIRFLOW_CTX_TASK_ID=get_name
AIRFLOW_CTX_EXECUTION_DATE=2021-11-25T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-11-25T00:00:00+00:00
[2022-07-29 11:41:36,156] {python.py:173} INFO - Done. Returned value was: None
[2022-07-29 11:41:36,175] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_python_operator, task_id=get_name, execution_date=20211125T000000, start_date=20220729T114135, end_date=20220729T114136
[2022-07-29 11:41:36,208] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-29 11:41:36,277] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
